misc/language-tour-guide/src/components/execution/History.tsx

```diff
@@ -73,8 +79,30 @@ export function History() {
                     </span>
                   </div>
                   <div className="rounded bg-white p-2 text-sm whitespace-pre-wrap">
+                    <div className="mb-1">
+                      <span className="text-grey-70 text-xs font-medium">
+                        Result:
+                      </span>
+                    </div>
                     {entry.result || "No result"}
                   </div>
+                  {entry.stdoutResult && entry.stdoutResult.length > 0 && (
+                    <div className="mt-2 rounded border-l-2 border-blue-400 bg-gray-50 p-2 text-sm whitespace-pre-wrap">
+                      <div className="mb-1 flex flex-col gap-1">
+                        <span className="text-grey-70 text-xs font-medium">
+                          Console output:
+                        </span>
+                        {entry.stdoutResult.map((line, index) => (
+                          <span
+                            key={`stdout-${index}-${line}`}
+                            className="text-brown-90/70 text-xs font-medium"
+                          >
+                            {line}
+                          </span>
+                        ))}
+                      </div>
+                    </div>
+                  )}
```

Comment:
Maybe it would be worth it to extract as a separate component with stdoutResult passed as property